src/jdk.jlink/share/classes/jdk/tools/jlink/internal/JlinkTask.java

```diff
@@ -81,6 +84,8 @@
 public class JlinkTask {
     public static final boolean DEBUG = Boolean.getBoolean("jlink.debug");
 
+    // Run time based link internal resources files
+    private static final String OTHER_RESOURCES_FILE = "jdk/tools/jlink/internal/runlink_%s_resources";
```

Comment:
Happy to change the prefix. Having a (unique) prefix seems useful, though, since it eases filtering. Note: that file is not only there for the hash sums, but also to keep track of the path to the files in the filesystem, which we'd otherwise not know.

Reply:
Will see how a few suggested renames looks like and a better term to describe this may come out 😄 `runlink` is a bit confusing.

Reply:
I just went with `fs` for `filesystem`. E.g. `fs_java.base_files` for `java.base`. Those files are taken from the filesystem so this naming makes some sense to me.